regexes/device/mobiles.yml

```diff
@@ -18784,14 +18848,17 @@ Maxtron:
 
 # Philco (philco.com.br)
 Philco:
-  regex: '(PCS01|PH350B|PTB7R)(?:[);/ ]|$)'
+  regex: 'PHILCO |(PCS01|PH350B|PTB7R)(?:[);/ ]|$)'
   device: 'smartphone'
   models:
     - regex: '(PCS01|PH350B)(?:[);/ ]|$)'
       model: '$1'
     - regex: '(PTB7R)(?:[);/ ]|$)'
       device: 'tablet'
       model: '$1'
+    - regex: 'PHILCO Android TV(?:[);/ ]|$)'
+      device: 'tv'
+      model: 'General Android TV'
```

Comment:
Not sure if should name such devices at all. Leaving the name empty would report them as `Unknown`, which would actually be true, as we don't know the exact model 🤔

Reply:
I will correct for an empty value